docs/porting/psa/crypto.md

```diff
@@ -0,0 +1,30 @@
+<h2 id="crypto-port">Mbed Crypto</h2>
+
+Mbed OS provides Mbed Crypto for targets that require support for entropy injection. For information about Mbed Crypto, please refer to [the Mbed Crypto repository](https://github.com/ARMmbed/mbed-crypto).

```

Comment:
```suggestion
Mbed Crypto library provides support for entropy injection. For information about Mbed Crypto, please refer to [the Mbed Crypto repository](https://github.com/ARMmbed/mbed-crypto).
```

Reply:
the same comment as above, those tests are part of mbed-os and not mbed-crypto. I think mbed-os is correct.